app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -195,7 +196,14 @@ def create_new_pvc(volume_name, capacity, pvc_name)
                          :requests      => { :storage => capacity + "G" })
     end
 
+    def delete_vm_pvcs(vm_uuid)
```

Comment:
the argument for this method should be the vm_name.
We're using (vm_name,namespace) pairs as unique identifier of the vm.
We do not rely on the vm-id since there is no guarantee it will be the consistent towards the life of the vm.